pkg/controller/worker/machines.go

```diff
@@ -175,6 +181,7 @@ func (w *workerDelegate) generateMachineConfig(ctx context.Context) error {
 				"disks":              disks,
 				"labels":             gceInstanceLabels,
 				"machineType":        pool.MachineType,
+				"nodeTemplate":       nodeTemplate,
```

Comment:
Why not keep things simple and create the whole `nodeTemplate` object right here?

Reply:
sure